lib/tunnel.js

```diff
@@ -0,0 +1,59 @@
+var net = require('net');
+var ssh2 = require('ssh2');
+
+module.exports = makeTunnel;
+
+function makeTunnel(urlObj, callback) {
+  var conn = new ssh2.Client();
+  var env = process.env;
+  var username = env.LOGNAME || env.USER || env.LNAME || env.USERNAME;
+
+  conn.on('ready', function() {
+    makeProxy(conn, urlObj, callback);
+    // don't keep the process alive if all we have is this connection
+    conn._sock.unref();
+  }).on('error', function(err) {
+    console.error('ssh connection error: ', err);
```

Comment:
can the error be emitted on whatever object this module returns, so LB could get the error and display in the UI?